.ci/update-github-page.sh

```diff
@@ -89,7 +89,7 @@ EOF
 echo "$JSON"
 
 echo "Updating Github tag page"
-curl \
+curl --fail-with-body \
```

Comment:
Shouldn't this be a requirement on all `curls` like we do with `e` for mvn? It seems to me we should want to fail on all curl failures, just that this was one example.
Example: https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_non_main_files_checks.xml#L26-L32

Reply:
@rnveach I agree, @stoyanK7 please create an issue for creation of new Regexp Singleline module. There might be some exceptions in case of retries (I think we have some scripts that try repeatedly to download something), we can deal with that in PR for new issue.

Reply:
Created
https://github.com/checkstyle/checkstyle/issues/12389

Let's unblock this PR